### PR TITLE
Add option to set ORT_DISABLE_ALL as optimization

### PR DESCRIPTION
**Pull request by casassg**
Please let me know if it would be better to use a different value instead of 2.

This change allows users to set optimization->graph->level to DISABLE_ALL by setting 2. The reason for this is that sometimes specially when using a triton server for serving multiple models, we prefer users to optimize the model offline instead of online. This allows us to disable optimization greatly speeding up load time into triton onnxruntime backend.

This is my first contribution here, so please do let me know if I should change anything or add more tests. Given the code is 2 lines, I didn't initially add it.

### CODE REVIEW

**Review by pranavsharma — APPROVED** [review]

**Comment by dyastremsky** [comment]
If you haven't already, do you mind to follow the contribution page [here](https://github.com/triton-inference-server/server/blob/main/CONTRIBUTING.md#contributor-license-agreement-cla) to submit a signed CLA?

Also, can you please update the [README documentation](https://github.com/triton-inference-server/onnxruntime_backend#model-config-options) for level to reflect this option?

CC: @tanmayv25

**Comment by dyastremsky** [comment]
@casassg Following up about the CLA and README updates.

**Comment by oandreeva-nv** [comment]
Hi @casassg, thank you for this PR and we are looking forward to merging it soon. However, we do need a signed CLA and a simple entry in the documentation about this level of optimization (for greater visibility of this feature). May I ask you if you have time in the nearest future to do these 2 actions?

**Comment by dyastremsky** [comment]
@casassg Following up about the CLA! If not, we can merge these in a separate PR on our own, if preferred.

Also, I am reading this a little closer and think it would be good to add a comment about this open in the README under [Model Config Options](https://github.com/triton-inference-server/onnxruntime_backend#model-config-options).

**Comment by nv-kmcgill53** [comment]
The triton project has accepted the CLA for Block Inc.. This PR should not be blocked on the CLA anymore.

**Comment by dyastremsky** [comment]
Fantastic, thank you Gerard and Kyle! @cassassg, would you be able to add the minor documentation updates, so we can get this merged? We can re-approve after.

**Comment by casassg** [comment]
@dyastremsky added a small comment (sorry this got deprioritized in my stack and buried down the line)

**Review by tanmayv25 — APPROVED** [review]

**Pull request merged**
